v2/workloadapi/addr.go

```diff
@@ -17,14 +17,9 @@ func GetDefaultAddress() (string, bool) {
 	return os.LookupEnv(SocketEnv)
 }
 
-func ValidateAddress(addr string) error {
-	_, err := parseTargetFromAddr(addr)
-	return err
-}
-
-// parseTargetFromAddr parses the endpoint address and returns a gRPC target
+// ParseTargetFromAddr parses the endpoint address and returns a gRPC target
 // string for dialing.
-func parseTargetFromAddr(addr string) (string, error) {
+func ParseTargetFromAddr(addr string) (string, error) {
```

Comment:
I'm ok with this being exported. But we should keep `ValidateAddress` as well.

Also, if we do export this, it should probably be consistent in naming with the other address related functions, i.e. `ParseTargetFromAddress`.

Reply:
No need. That change was a mistake on my part. I will make it work without needing to export it. Thanks!